src/test/libcephfs/flock.cc

```diff
@@ -53,9 +53,8 @@
 static const mode_t fileMode = S_IRWXU | S_IRWXG | S_IRWXO;
 
 // Default wait time for normal and "slow" operations
-// (5" should be enough in case of network congestion)
-static const long waitMs = 10;
-static const long waitSlowMs = 5000;
+static const long waitMs = 1 * 1000;
+static const long waitSlowMs = 30 * 1000;
```

Comment:
I think large timeout value will reduce the possibility of random failure. I remove that commit since you don't like it.